docs/Documentation.adoc

```diff
@@ -71,10 +71,6 @@ If set, the name will be displayed near the top of the page.
 Setting this will add a "View on GitHub" link in the navigation bar.
 |_not set_
 
-|`site-seedu`
-|Define this attribute if the project is an official SE-EDU project.
-This will render the SE-EDU navigation bar at the top of the page, and add some SE-EDU-specific navigation items.
-|_not set_
```

Comment:
Haha so this removes SE-EDU links from our website?

Reply:
Not sure